internal/stringutils/stringutils.go

```diff
@@ -0,0 +1,6 @@
+//go:build !windows
+
+package stringutils
+
+// RemoveCR by default does nothing, it only modifies the string if built for Windows.
```

Comment:
```suggestion
// RemoveCR by default does nothing for building in  other than Windows OS.
```
I was bit confused by this comment, firstly I thought that you've missed something or the flag is wrong.

Reply:
I tried making it more explicit, please check the updated version.